frontstage/cloud/cloudfoundry.py

```diff
@@ -12,18 +12,6 @@ def __init__(self):
     def detected(self):
         return self._cf_env.app
 
-    @property
-    def port(self):
-        return self._cf_env.port
-
-    @property
-    def host(self):
-        return self._host
-
-    @property
-    def protocol(self):
-        return self._protocol
-
```

Comment:
Why have you deleted this?

Reply:
we don't use it

Reply:
The variable names indicate they're 'private' so to access them the property methods should be used